pkg/ui/src/views/statements/statementDetails.tsx

```diff
@@ -0,0 +1,252 @@
+
+interface NumericStatTableProps {
+  rows: NumericStatRow[];
+  count: number;
+  format: (v: number) => string;
+}
+
+class NumericStatTable extends React.Component<NumericStatTableProps> {
+  static defaultProps = {
+    format: (v: number) => `${v}`,
+  };
+
+  render() {
+    return (
+      <table className="numeric-stats-table">
+        <thead>
+          <tr className="numeric-stats-table__row--header">
+            <th className="numeric-stats-table__cell" />
+            <th className="numeric-stats-table__cell">Mean</th>
+            <th className="numeric-stats-table__cell">Std. Dev.</th>
+          </tr>
+        </thead>
+        <tbody style={{ textAlign: "right" }}>
+          {
+            this.props.rows.map((row: NumericStatRow) => {
+              return (
+                <tr className="numeric-stats-table__row--body">
+                  <th className="numeric-stats-table__cell" style={{ textAlign: "left" }}>{ row.name }</th>
+                  <td className="numeric-stats-table__cell">{ this.props.format(row.value.mean) }</td>
+                  <td className="numeric-stats-table__cell">{ this.props.format(stdDev(row.value, this.props.count)) }</td>
+                </tr>
+              );
+            })
+          }
+        </tbody>
+      </table>
+    );
+  }
+}
+
+class StatementDetails extends React.Component<StatementDetailsProps> {
+  componentWillMount() {
+    this.props.refreshQueries();
+  }
+
+  componentWillReceiveProps() {
+    this.props.refreshQueries();
+  }
+
+  render() {
+    return (
+      <div>
+        <Helmet>
+          <title>Details | Statements</title>
+        </Helmet>
+        <section className="section">
+          <h2>Statement Details</h2>
+          <Loading
+            loading={_.isNil(this.props.statement)}
+            className="loading-image loading-image__spinner"
+            image={spinner}
+          >
+            { this.renderContent() }
+          </Loading>
+        </section>
+      </div>
+    );
+  }
+
+  renderContent() {
+    if (!this.props.statement) {
+      return null;
+    }
+
+    const { stats } = this.props.statement;
+
+    const count = FixLong(stats.count).toInt();
+    const firstAttemptCount = FixLong(stats.first_attempt_count).toInt();
+
+    const countBar = countBarChart();
+    const latencyBar = latencyBarChart();
+
+    return (
+      <div className="content l-columns">
+        <div className="l-columns__left">
+          <section className="section">
+            <SqlBox value={ this.props.statement.key.query } />
+          </section>
+          <section className="section">
+            <table className="numeric-stats-table">
+              <tbody>
+                <tr className="numeric-stats-table__row--body">
+                  <th className="numeric-stats-table__cell" style={{ textAlign: "left" }}>App</th>
+                  <td className="numeric-stats-table__cell" style={{ textAlign: "right" }}>{ this.props.statement.key.app }</td>
```

Comment:
these are hashed server-side before they're returned: https://github.com/cockroachdb/cockroach/blob/dc15c475de05cfde4dabda55c6823d358b5db585/pkg/sql/app_stats.go#L322

![image](https://user-images.githubusercontent.com/7341/41608120-b6ae0302-73b5-11e8-8b8b-0c4927e3b05d.png)

I agree that this is a valuable column to have, but suggest leaving it out and figuring out a way to get the unhashed name in a later PR. (Plus, a widget letting you see and manipulate the filters would be good)